### PR TITLE
fix getNonLocalizedAttributes route for ST

### PR DESCRIPTION
**Pull request by petersg83**
It was throwing a 404 for a locale different from the default locale

### CODE REVIEW

File: packages/strapi-plugin-i18n/controllers/content-types.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ module.exports = {
       return ctx.badRequest('model.not.localized');
     }
 
-    let params = modelDef.kind === 'singleType' ? {} : { id };
+    let params = modelDef.kind === 'singleType' ? { _locale: 'all' } : { id };
```
Comment (alexandrebodin) [inline packages/strapi-plugin-i18n/controllers/content-types.js:34]: why not `_locale: locale` ?

Comment (petersg83) [reply packages/strapi-plugin-i18n/controllers/content-types.js:34]: `locale` is the locale that doesn't exist yet and that the front would like to create.
The found entity is only used to get the `nonLocalizedAttributes`. With the CT we have the `id` of a related entity but not with the ST and any will do.

Comment (petersg83) [reply packages/strapi-plugin-i18n/controllers/content-types.js]: Good idea, that works too

**Review by alexandrebodin — APPROVED** [review]
that works :D

**Pull request merged**